src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-card-pop-up/ubs-admin-tariffs-card-pop-up.component.ts

```diff
@@ -313,10 +316,10 @@ export class UbsAdminTariffsCardPopUpComponent implements OnInit, OnDestroy {
     let englishLocation;
     let locationId;
     event.option.value.locationTranslationDtoList.forEach((el) => {
-      if (el.languageCode === 'ua') {
+      if (el.languageCode === Language.UA) {
         location = el.locationName;
       }
-      if (el.languageCode === 'en') {
+      if (el.languageCode === Language.EN) {
```

Comment:
You can replace two if by - `el.languageCode === Language.UA ? location = el.locationName : englishLocation = el.locationName`